man/ocamlopt.m

```diff
@@ -728,6 +728,19 @@ The AMD64 code generator (64-bit versions of Intel Pentium and AMD
 .B \-fno\-PIC
 Generate position-dependent machine code.
 
+.SH OPTIONS FOR THE POWER ARCHITECTURE
+
+The PowerPC code generator supports the following additional options:
+.TP
+.B \-flarge\-toc
+Enables the PowerPC large model allowing the TOC (table of contents) to be
+greater than 64 kbytes.  This is the default since 4.11.
+.TP
+.B \-fsmall\-toc
+Enables the PowerPC small model allowing the TOC to be up to 64 kbytes.  Prior
```

Comment:
Technically, it's 64 kbytes **per compilation unit**.  But I'm not sure we need to be that precise.